src/task/CreatePeerConnectionTask.ts

```diff
@@ -83,17 +83,12 @@ export default class CreatePeerConnectionTask extends BaseTask implements Remova
     configuration.bundlePolicy = this.context.browserBehavior.requiresBundlePolicy();
     // @ts-ignore
     configuration.sdpSemantics = 'unified-plan';
-    // @ts-ignore
-    this.logger.info(`SDP semantics are ${configuration.sdpSemantics}`);
-    const connectionConstraints = {
```

Comment:
Would it be good to document these constraint removal in changelog? It currently just says deprecated constraints. Also the same link from description can also be good addition to changelog for this I think.